server/schemas/thread.graphql

```diff
@@ -0,0 +1,49 @@
+type Thread {
+  id: ID!
+  workspace: Workspace
+  workspaceId: ID!
+  comments: [Comment!]!
+}
+
+type Comment {
+  id: ID!
+  author: User
+  authorId: ID!
+  content: String!
+  createdAt: DateTime!
+}
+
+input CreateCommentInput {
+  threadId: ID!
+  authorId: ID!
```

Comment:
Do we need the `authorId` here? cant we use the logged-in user id from `operator`

Reply:
@yk-eukarya I think that we do not need authorId here, this applies to `UpdateCommentInput` as well right?

Reply:
Done!